sources/packages/backend/apps/queue-consumers/src/processors/schedulers/esdc-integration/msfaa-integration/_tests_/msfaa-part-time-process-integration.scheduler.models.ts

```diff
@@ -0,0 +1,84 @@
+import { OfferingIntensity, RelationshipStatus } from "@sims/sims-db";
+
+/**
+ * Values part of an MSFAA file record.
+ */
+export interface MSFAATestInputData {
+  msfaaNumber: string;
+  sin: string;
+  institutionCode: string;
+  birthDate: string;
+  surname: string;
+  givenName: string;
+  gender: string;
+  maritalStatus: RelationshipStatus;
+  addressLine1: string;
+  addressLine2: string;
```

Comment:
Looking at the file-details, few of these inputs could be optional right? May be that could become a scenario in future if needed

![image](https://user-images.githubusercontent.com/54600590/234989172-bd8cb0e2-25e2-4807-a27b-7b8fb9dcd518.png)

Reply:
They can be test scenarios but not necessarily E2E scenarios. The idea is to have some E2E and focus more on the unit tests for the queue consumers, as per my understanding, to compensate also the decision to have E2E running sequentially.